deepchem/data/data_loader.py

```diff
@@ -1012,7 +1012,7 @@ def _add_sequence(sequences: np.ndarray,
           # TODO log attempts to add empty sequences every shard
           return np.array([])
         # Annotate start/stop of sequence
-        if auto_add_annotations:
+        if self.auto_add_annotations:
```

Comment:
Is `auto_add_annotations` set in the constructor? If so, could you add a unit test that would have caught this bug so we can make sure we don't repeat the bug in the future?

Reply:
I'm on it!